reactfx/src/main/java/org/reactfx/DefaultEventStream.java

```diff
@@ -0,0 +1,43 @@
+package org.reactfx;
+
+import java.util.function.Consumer;
+
+public class DefaultEventStream<T> extends EventStreamBase<T> {
```

Comment:
Right. I forgot to check the other classes that you wrote as an example for this area.
It's fixed in the next commit.